main.go

```diff
@@ -50,8 +52,11 @@ var (
 	enableLeaderElection = flag.Bool("leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	enableProfile = flag.Bool("enable-pprof", false, "enable pprof profiling")
-	profilePort   = flag.Int("pprof-port", 6060, "port for pprof profiling. defaulted to 6060 if unspecified")
+	enableProfile       = flag.Bool("enable-pprof", false, "enable pprof profiling")
+	profilePort         = flag.Int("pprof-port", 6060, "port for pprof profiling. defaulted to 6060 if unspecified")
+	eraserPullPolicy    = flag.String("eraser-pull-policy", string(corev1.PullIfNotPresent), "pull policy for eraser image")
```

Comment:
It's complicated because the configuration for the eraser/scanner/collector containers are set in code, and not in the helm chart. For example to get args to the scanner from the helm chart we have to set --scanner-arg on the manager.

Reply:
The tests use `--set controllerManager.image.additionalArgs={...}` to propagate these values to the right place. Maybe we should consider a system-wide configmap that the manager looks at when it boots up, so that we don't clutter up the arguments like this? @sozercan if you agree I'll open an issue for that.

Reply:
I am okay with configmap then cluttering arguments (which will get worse over time). Can we fix the e2e issue first then come up with a design for configmap solution?

Reply:
The configmap issue is being tracked by #446. I'm addressing the e2e issues